develop/dev-guide-build-cluster-in-cloud.md

```diff
@@ -27,19 +27,20 @@ This document walks you through the quickest way to get started with TiDB Cloud.
 
 2. [Log in](https://tidbcloud.com/) to your TiDB Cloud account.
 
-3. On the [**Clusters**](https://tidbcloud.com/console/clusters) page, click **Create Cluster**.
+    The [**Clusters**](https://tidbcloud.com/console/clusters) list page is displayed by default.
 
-4. On the **Create Cluster** page, **Serverless Tier** is selected by default. Update the default cluster name if necessary, and then select the region where you want to create your cluster.
+3. For the first-time login, TiDB Cloud creates a default Serverless Tier cluster `Cluster0` for you automatically. You can either use this default cluster for the subsequent steps or create a new Serverless Tier on your own.
 
-5. Click **Create** to create a Serverless Tier cluster.
+    To create a new Serverless Tier on your own, take the following operations:
 
-    Your TiDB Cloud cluster will be created in approximately 30 seconds.
+    1. Click **Create Cluster**.
+    2. On the **Create Cluster** page, **Serverless Tier** is selected by default. Update the default cluster name if necessary, select a target region of your cluster, and then click **Create**. Your Serverless Tier cluster will be created in approximately 30 seconds.
 
-6. After your TiDB Cloud cluster is created, click your cluster name to go to the cluster overview page, and then click **Connect** in the upper-right corner. A connection dialog box is displayed.
+4. Click the target cluster name to go to its overview page, and then click **Connect** in the upper-right corner. A connection dialog box is displayed.
```

Comment:
```suggestion
4. After your cluster is created, click the target cluster name to go to its overview page, and then click **Connect** in the upper-right corner. A connection dialog box is displayed.
```
<img width="516" alt="image" src="https://user-images.githubusercontent.com/60599231/212868540-8d590483-8dfc-4dd0-ba27-d7b75b726bf5.png">
The "cluster name" is not clickable during the cluster creation process.